autogpts/forge/forge/components/user_interaction/user_interaction.py

```diff
@@ -1,19 +1,17 @@
 from typing import Iterator
 
-from autogpt.agents.protocols import CommandProvider
-from autogpt.app.utils import clean_input
-from autogpt.command_decorator import command
-from autogpt.config.config import Config
-from autogpt.core.utils.json_schema import JSONSchema
-from autogpt.models.command import Command
-from autogpt.utils.utils import DEFAULT_ASK_COMMAND
+from forge.agent.protocols import CommandProvider
+from forge.command import Command, command
+from forge.config.config import Config
+from forge.json.schema import JSONSchema
+from forge.utils.const import DEFAULT_ASK_COMMAND
+from forge.utils.input import clean_input
 
 
 class UserInteractionComponent(CommandProvider):
     """Provides commands to interact with the user."""
 
     def __init__(self, config: Config):
-        self.config = config
```

Comment:
IMO this should be part of CommandProvider and be called by a `super().__init__()` rather than having every instance manually handle it. (this is a later problem though). I'd say revert this line for now unless it throws an error

Reply:
Not sure what you mean but `AgentComponent`s should have their own configuration and I have this on my todo list. It isn't necessary to cache the `config` so I removed it.